src/app/app.component.scss

```diff
@@ -0,0 +1,17 @@
+.custom {
+  width: 50%;
+  text-align: center;
+}
+.formats {
+  width: 100%;
+  padding: 10px;
+}
+.btn {
+  padding: 10px;
+  margin-top: 5px;
+  background-color: #eee;
```

Comment:
colors are better declared and used as scss variables

Reply:
To my mind it's optional now.
We can eject color variables, fonts, etc. when we will define design of app.

Reply:
i removed these styles, because they were made for testing, and do not make sense now